### DISCUSSION TITLE
[Feat]: Compilation and networking

### DISCUSSION THREAD

**Discussion by chenhuanguang** [coder/code-server#5646] [Q&A]
I want to know how to recompile the modified code into an executable file after getting the entire code base; Also, do I need a network to start the server and open the webpage version of vscode

**Comment by jsjoeio** [comment]
You should be able to follow [this](https://github.com/coder/code-server/blob/main/docs/CONTRIBUTING.md#development-workflow) to build it to an executable. No, you can do it all locally on your machine.
